JEduardoRJx/palette-picker-frontend
# Setup react

What’s this PR do?
This is just the very basic react app with the dependencies installed
#### Where should the reviewer start?
look at package.json()
#### How should this be manually tested?
Run `npm start` in your terminal with this branch pulled down and open the server to see react app. 
#### Any background context you want to provide?
NA
#### What are the relevant tickets?
NA
#### Screenshots (if appropriate)
NA
#### Questions:
